### PR TITLE
Enable auto updates for WCF branch 2.1.0

### PR DESCRIPTION
**Pull request by StephenBonikowsky**

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
@dagood @MattGal could one of you review please?

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -850,6 +850,35 @@
         }
       }
     },
+    // Update dependencies in WCF release/2.1.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectn-tfs/master/Latest.txt",
```
Comment (dagood) [inline Maestro/subscriptions.json]: This one was turned off in CoreFX: https://github.com/dotnet/corefx/blob/c61475013bc71f6d33f9da2694437adccb125588/dependencies.props#L96. (Don't know for sure if this needs to be matched in WCF though.)

Comment (StephenBonikowsky) [reply Maestro/subscriptions.json]: I believe it is used to get the latest version of the test ILC used for UWP runs. Is CoreFx not doing any UWP runs anymore?

Comment (MattGal) [reply Maestro/subscriptions.json]: Yes, as Davis noted they were turned off, I think some stuff got broken and it will be a while til it's fixed.

Comment (StephenBonikowsky) [reply Maestro/subscriptions.json]: Ok, will remove it.

Comment (dagood) [reply Maestro/subscriptions.json]: Make sure to change it in WCF, since this file only controls when updates are triggered.

**Pull request merged**